src/schema/mutation/editDashboard.mutation.ts

```diff
@@ -77,6 +118,16 @@ export default {
         buttons?: any;
         gridOptions?: any;
       } = {};
+
+      // Update URL to base64 file
+      for (const [index, struct] of args.structure.entries()) {
```

Comment:
@morgang-adappt @HasnatL-Adapptlimited 

We have the feeling that this is an artifact from previous iterations and that it is not used
Is that correct?

Reply:
Hey @AntoineRelief This is still used 

The only way svg images that come from a url can be exported i.e. https://whoemssafepsta02.blob.core.windows.net/hazards/chemical.svg is if you insert it and refresh the page, this snippet of code converts it to base64 then blob

Without this snippet images like the logos you see in event cards, can't be exported
After inserting to text widget and refreshing page
![image](https://github.com/user-attachments/assets/c8a059a5-2f10-4f7b-bccd-8d67f6db5314)

![image](https://github.com/user-attachments/assets/d56c548a-83f8-4dae-b458-8e6a4b73b028)